### DISCUSSION TITLE
volume: -max is ignored

### DISCUSSION THREAD

**Discussion by kw005** [seaweedfs/seaweedfs#2143] [General]
Coming from 0.76 with 6 volumes and migrated to Version 2.27, everything is working fine so far.
However, starting the volume servers with (max=12 or max=1). Upload directory assigning is always from 1...6, instead randomly 1...12 or 1 only when setting max=1

Am I missing something or why is this setting ignored?

Thanks for any help.

**Comment by chrislusf** [comment]
The volumes are only allocated when current ones are exhausted.
You can https://github.com/chrislusf/seaweedfs/wiki/Master-Server-API#pre-allocate-volumes
